bibcheck/rules.cfg

```diff
@@ -24,3 +24,12 @@ filter_collection = HEP
 
 [check_earliest_date]
 check = earliest_date
+
```

Comment:
The code is fully tested. mvtexkey_z2a will touch MANY!! records. To kow how many I have to run and count instead of changing.